README.md

```diff
@@ -229,6 +230,14 @@ recorded values will not be recalculated if this value is changed. If you would
 like to change previous values, you would need to write a migration of the data
 to recalculate history.
 
+## Creating accounts
+
```

Comment:
Could we also add in a bit here that describes how _by default_ conbench has open read access, so a user account is not required (even for API access).

But if you have a private conbench instance or you're attempting to write data to conbench, a user account is required.